modular_chomp/code/modules/mob/living/simple_mob/subtypes/vore/plants/pitcher.dm

```diff
@@ -292,10 +307,11 @@ GLOBAL_LIST_INIT(pitcher_plant_lure_messages, list(
 	var/datum/seed/seed = null
 	var/obj/item/seeds/pit = null
 
-/obj/item/reagent_containers/food/snacks/pitcher_fruit/Initialize(mapload)
+/obj/item/reagent_containers/food/snacks/pitcher_fruit/Initialize()
 	. = ..()
 	reagents.add_reagent(REAGENT_ID_PITCHERNECTAR, 5)
-	bitesize = 4
+	reagents.add_reagent(REAGENT_ID_PARALYZE_FLUID, 5) // Something worth harvesting the fruits for.
```

Comment:
This is the new reagent. I'm open to other ideas if you don't want this, but I want something players will actually seek out and have an incentive to harvest.